tx/transaction_test.go

```diff
@@ -17,14 +17,16 @@ import (
 	"github.com/vechain/thor/v2/tx"
 )
 
-func GetMockTx() tx.Transaction {
+func GetMockTx(txType int) tx.Transaction {
```

Comment:
annoyingly, what if these tests were tabled-test (something like `for _,txTypes range []int{tx.LegacyTxType, tx.DynamicFeeTxType}` for the 2 tx types ? Would that add value ?

Reply:
yes, I'll change those that are prone to this approach